recipes/openssl/3.x.x/conanfile.py

```diff
@@ -650,6 +653,11 @@ def package(self):
                 if file.endswith(".a"):
                     os.unlink(os.path.join(libdir, file))
 
+
+        if not self.options.no_fips:
+            provdir = os.path.join(self._source_subfolder, "providers")
+            self.copy("fips.so", src=provdir,dst="lib/ossl-modules")
```

Comment:
will it be fips.dll or fips.dylib on other platforms?

Reply:
Updated to copy fips lib based on os.